### ISSUE TITLE
Cannot find ADE key

### ISSUE THREAD

**Issue by Peter-Harris** [apprenticeharper/DeDRM_tools#616]
When customising the DeDRM plugin I cannot add an ADE key, I get the message "The default encryption key for Adobe Digital Editions could not be found". I am running Calibre 2.55 in Ubuntu 16.04 with ADE running in Wine.

**Comment by ElleKayEm** [comment]
There's been a long standing issue where DeDRM can't run the key script because it can't find Python under Wine.  Some discussion in issue #53 and other threads.

**Comment by ElleKayEm** [comment]
Yes, the Obok plugin also available here works with Kobo Desktop.
